plutus-ledger-api/src/v1/interval.rs

```diff
@@ -247,6 +384,52 @@ where
     PosInf,
 }
 
+impl<T> Ord for Extended<T>
+where
+    T: Ord,
+{
+    fn cmp(&self, other: &Self) -> cmp::Ordering {
+        match self {
```

Comment:
Why not: `match (self, other)`? Also, what's the reason to implement both `PartialOrd` and `Ord` manually?

Reply:
good idea